components/Footer.vue

```diff
@@ -1,10 +1,44 @@
 <template>
-    <div class="flex bg-teal-700 text-white">
-        <div class="flex-auto">
-            <a href="https://www.netlify.com/" target="_blank">Deployed on Netlify</a>
+    <div class="footer flex bg-orange-100 text-sm text-current font-semibold py-2">
+        <div class="left-footer-section flex flex-0 justify-start">
+            <NuxtLink to="/" class="self-center ml-8 mr-2">
+                © 2023 Find a doc JP
```

Comment:
Nit: Let's make an issue to add a function so the year is always up to date?

Also, the official name is: `Find a Doc, Japan`

Reply:
ah good idea! I just copied from the mockup XD

Reply:
updated